regulations/make.py

```diff
@@ -210,7 +211,12 @@ def buildLanguage(args, language):
 
   pdfName = languageData[language]["pdf"]
 
-  html.html(language, buildDir, pdfName + ".pdf", isTranslation=isTranslation, verbose=args.verbose)
+  srcDir = "translations/" + language if isTranslation else "wca-regulations"
+
+  warnings = check_missing.check_guidelines(language, srcDir)
+  if len(warnings) > 0:
+    print warnings.encode('utf-8')
```

Comment:
I don't like how this encode leaked out to here, would this go away if we have `check_missing.check_guidelines` return an array of (encoded) strings instead?